drape_frontend/apply_feature_functors.cpp

```diff
@@ -891,9 +869,9 @@ void ApplyLineFeatureGeometry::ProcessRule(LineRuleProto const & lineRule)
 ApplyLineFeatureAdditional::ApplyLineFeatureAdditional(TileKey const & tileKey, TInsertShapeFn const & insertShape,
                                                        FeatureType & f, double currentScaleGtoP,
                                                        CaptionDescription const & captions,
-                                                       std::vector<m2::SharedSpline> const & clippedSplines)
+                                                       std::vector<m2::SharedSpline> clippedSplines)
```

Comment:
Why not use && to avoid copying if it is always moved in the code?

Reply:
This is a common good pattern, you can pass const ref of rvalue ref.

Reply:
https://godbolt.org/z/rEodo9e9c

Reply:
godbolt inlined everything and called additional dtor.

Yes, one more empty object to dctor, but more flexible to use. Ok, will add &&